src/codecoverage/bot/code_coverage_bot/taskcluster.py

```diff
@@ -20,7 +20,7 @@ def get_task(branch, revision, platform):
     if platform == 'linux':
         platform_name = 'linux64-ccov-debug'
         product = 'firefox'
-    elif platform == 'win':
+    elif platform == 'windows':
```

Comment:
The platform name changed ?

Reply:
This was just an internal name used in the task, I've changed it to `windows` as that's the name used in artifacts.py and so that in https://github.com/mozilla/release-services/pull/1756/files#diff-8b7a33038648c10b3bacdf6e49c8bffdR78 I could just have a list and not a list of tuples